src/reducers/query.ts

```diff
@@ -87,16 +87,32 @@ export default {
   query: handleActions(
     {
       [ActionTypes.PREVIEW_DATA_SUCCESS]: (state, action: any) => {
-        const rows = action.payload.queryResults.data.result;
         const columnsByName = _.keyBy(state.columns, 'name');
-        const columns = action.payload.columns.map((column: ColumnModel) => ({
+        const columns: TableColumnType[] = action.payload.columns.map((column: ColumnModel) => ({
           name: column.name,
           dataType: column.datatype,
           arrayOf: column.array_of,
           allowSort: !column.array_of,
           allowResize: true,
           width: columnsByName[column.name]?.width || TABLE_DEFAULT_COLUMN_WIDTH,
         }));
+        // We only need to re-format row data of type timestamp
```

Comment:
Previously, we also reformatted data of type "integer" and "float". I no longer think there is need to do this.

I believe the main reason was to round to 3 significant digits so that numbers would be displayed without overflow in the UI. However, our new(er) LightTable component already handles overflow well. Additionally, we have had a user specifically request for all significant digits to be displayed in the UI.